apheleia.el

```diff
@@ -690,6 +901,8 @@ formatter being run, for diagnostic purposes."
                  (unwind-protect
                      (funcall callback scratch)
                    (kill-buffer scratch)))
+               ;; A metadata field
+               `((remote . ,remote))
```

Comment:
Is this changing the expected interface of Elisp-based formatters? If so we would want to update the docstring of `apheleia-formatters` and announce a breaking change.

If we are making a breaking change anyway, it might be a good time to make the interface extensible, by saying that the function must receive its parameters using keyword arguments, and ignore any unknown keys.

Reply:
@raxod502 

It looks like functional formatters are still unreleased (in the CHANGELOG), so is a breaking change still necessary?